examples/arm/aot_arm_compiler.py

```diff
@@ -38,6 +38,34 @@ def forward(self, x):
     can_delegate = True
 
 
+class AddModule2(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x, y):
+        return x + y
+
+    example_input = (
+        torch.ones(5, dtype=torch.int32),
+        torch.ones(5, dtype=torch.int32),
+    )
+    can_delegate = True
+
+
+class AddModule3(torch.nn.Module):
```

Comment:
can we add something with weights? I was trying Linear and it was failing with an error in vela

Reply:
It's on the list, but i'd aim to do this separately in an "improve vela" set of changes.